src/main/features/response/views/evidence.njk

```diff
@@ -12,32 +12,31 @@
 {% block content %}
 <div class="grid-row">
   <div class="column-two-thirds">
-
-    <div class="description-block">
-      <h3 class="heading-small">Their evidence</h3>
+    {% if evidence.rows %}
```

Comment:
Can combine the ifs

Reply:
if someone doesn't supply evidence is there an empty object or undefined?

Reply:
Fixed!